build.gradle.kts

```diff
@@ -2,23 +2,21 @@ plugins {
     groovy
     `java-gradle-plugin`
     `build-scan`
+    com.playframework.gradle.`test-setup`
```

Comment:
I'd assume that's the preferred syntax for applying "project-internal" plugins? Is there a benefit over applying it with the `id` syntax?

Reply:
Yes. The benefit is static compilation, you'll get a nice compiler message if you change a plugin id. When you add new plugin id, you'll get completion to add them.

Reply:
OK, thanks. Maybe I didn't find this fact in the documentation or the benefits are not clearly layed out in the documentation. It's probably worth to make the statement stand out as it has clear benefits.

Generally speaking, as a user I'd prefer a single way to apply plugins. It's just hard to keep all the reasons in your head why you should apply a plugin in different ways.